src/GrpcCredentialsHelper.php

```diff
@@ -32,51 +32,83 @@
 namespace Google\GAX;
 
 use Google\Auth\ApplicationDefaultCredentials;
+use Google\Auth\CredentialsLoader;
+use Google\Auth\FetchAuthTokenCache;
+use Google\Auth\Cache\MemoryCacheItemPool;
 use Grpc\ChannelCredentials;
 
 /**
- * Class that provides helpful utility functions for bootstrapping a gRPC client.
+ * A class that manages credentials for an API object using the Google Auth library
  */
-class GrpcBootstrap
+class GrpcCredentialsHelper
 {
+    private $scopes;
+    private $credentialsLoader;
+
     /**
-     * Provides a default instance of GrpcBootstrap.
+     * Accepts an optional credentialsLoader argument, to be used instead of using
+     * the ApplicationDefaultCredentials
+     *
+     * @param string[] $scopes A list of scopes required for API access
+     *
+     * @param array $optionalArgs {
+     *     Optional arguments. Arguments in addition to those documented below
+     *     will be passed as optional arguments to Google\Auth\FetchAuthTokenCache
+     *     when caching is not disabled.
+     *
+     *     @var Google\Auth\CredentialsLoader $credentialsLoader
+     *          A user-created CredentialsLoader object. Defaults to using
+     *          ApplicationDefaultCredentials
+     *     @var boolean $disableCaching
+     *          Disable caching of access tokens. Defaults to false
```

Comment:
It's easier to understand when you have a positive wording instead of a double negative. Thus I would recommend $enableCaching which defaults to true.